windows/src/developer/TIKE/compile/MergeKeyboardInfo.pas

```diff
@@ -374,6 +373,26 @@ procedure TMergeKeyboardInfo.AddName;
   json.AddPair('name', FName);
 end;
 
+//
+// isRTL -- from js
```

Comment:
Is there a reason to regex the .js instead of using the line in .kmn?

```
store(&KMW_RTL) '1'
```

It's tied to the "Keyboard is right-to-left" checkbox in Developer, right?

Reply:
We don't have access to the .kmn at this point. So sadly, yes, this is the only way to do it at present.